api/view/static/openapi.json

```diff
@@ -0,0 +1,268 @@
+            }
+          }
+        }
+      }
+    },
+    "/geometry/{dataset_geomId}": {
+      "get": {
+        "tags": [
+          "geometry"
+        ],
+        "summary": "Get geometry instance",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "text/plain": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/turtle": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "application/geo+json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "getGeometryByDatasetIdGeomId",
+        "parameters": [
+          {
+            "name": "dataset_geomId",
+            "in": "path",
+            "description": "Dataset and Geometry ID",
+            "required": true,
+            "schema": {
+              "type": "string"
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instance",
+        "description": "",
+        "responses": {
+          "200": {
+            "description": "OK",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "string"
+                }
+              },
+              "text/html": {
+                "schema": {
+                  "type": "string"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "searchGeomByLatLng",
+        "parameters": [
+          {
+            "name": "x",
+            "in": "path",
+            "description": "x",
+            "required": true,
+            "schema": {
+              "type": "number",
+              "example": 142.111
+            }
+          },
+          {
+            "name": "y",
+            "in": "path",
+            "description": "y",
+            "required": true,
+            "schema": {
+              "type": "number",
+              "example": -32.1231
+            }
+          },
+          {
+            "name": "crs",
+            "in": "query",
+            "description": "crs",
+            "required": false,
+            "schema": {
+              "type": "number"
+            }
+          }
+        ]
+      }
+    },
+    "/search/latlng/{x},{y}/dataset/{dataset_id}": {
+      "get": {
+        "tags": [
+          "search"
+        ],
+        "summary": "Get geometry instance",
```

Comment:
Ah - here we introduce the notion of 'dataset' in a meaningful way. What does it mean really?

Reply:
the intention was to provide a filter by the dataset. you could view a dataset as a collection of sorts generally. the use case is: tell me which geometries contain the input position that is a member of dataset.